### PR TITLE
python3Packages.okta: 2.9.2 -> 2.9.3

### PR DESCRIPTION
**Pull request by jbgosselin**
## Description of changes

- python3Packages.okta: 2.9.2 -> 2.9.3 (with fixed aenum dependency)

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jbgosselin** [comment]
@fabaff This can be merged before #241853 So it doesn't break functionality.

File: pkgs/development/python-modules/okta/default.nix [hunk 1]
```diff
@@ -20,20 +20,30 @@
 , yarl
 }:
 
+let
+  aenum_fixed = aenum.overridePythonAttrs (oldAttrs: rec {
```
Comment (fabaff) [inline pkgs/development/python-modules/okta/default.nix:24]: Overriding in `python-modules` should not be done. The proper fix would be to get support for `aenum` > 3.1.11 in `okta`.

https://github.com/okta/okta-sdk-python/issues/368

Comment (jbgosselin) [reply pkgs/development/python-modules/okta/default.nix:24]: I took the initiative to make a PR that would fix their package https://github.com/okta/okta-sdk-python/pull/370

**Comment by fabaff** [comment]
Was updated in https://github.com/NixOS/nixpkgs/pull/251878

**Pull request closed without merging**